tests/test_stream.py

```diff
@@ -476,6 +478,19 @@ class CustomWriter(CSVWriter): pass
             (2, ['id', 'name'], ['1', 'english']),
             (3, ['id', 'name'], ['2', '中国人'])]
 
+def test_stream_save_custom_writers1(tmpdir):
```

Comment:
Please rename the test (custom_writers1 -> json) and rebase on the full usage of `tmpdir` as I showed here - https://github.com/frictionlessdata/tabulator-py/pull/273#issuecomment-541666453 - for example. So it will not need to create additional files in the data dir